PocketKit/Sources/PocketKit/Article/ReadableViewModel/SavedItemViewModel.swift

```diff
@@ -123,8 +123,22 @@ class SavedItemViewModel: ReadableViewModel {
         }
 
         Task {
-            try? await self.source.fetchDetails(for: self.item)
+            do {
+                let remoteHasArticle = try await self.source.fetchDetails(for: self.item)
```

Comment:
Consider adding a breadcrumb here of what we are trying to fetch.
Then we will have more context if the catch hits.

Reply:
there's a breadcrumbs added in `fetchDetails`, do we find more value to add another, if so what details would you suggest?

 Log.breadcrumb(category: "recommendations", level: .debug, message: "Loading details for Recomendation: \(String(describing: recommendation.remoteID))")
 
Log.breadcrumb(category: "sync", level: .debug, message: "Fetching details for item with id \(String(describing: savedItem.remoteID))")